src/config/mod.rs

```diff
@@ -329,10 +329,9 @@ impl<'c> Builder<'c> {
         self
     }
 
-    /// Set the jailer process configuration.
-    pub fn jailer_cfg(mut self, jailer_cfg: Option<Jailer<'c>>) -> Self {
-        self.0.jailer_cfg = jailer_cfg;
-        self
+    /// Create the jailer process configuration builder.
+    pub fn jailer_cfg(self) -> JailerBuilder<'c> {
```

Comment:
not sure if we should rename the method to have "builder" in the name

Reply:
the more I think about this, the more I think `_builer` suffix won't bring anything. So I'll go w/o it for now and let's think about renaming later if you disagree?